src/terrain/clients/bags.clj

```diff
@@ -46,3 +46,19 @@
   [username id]
   (:body (http/delete (bags-url [username id])))
   nil)
+
+(defn get-default-bag
+  [username]
+  [username]
+  (:body (http/get (bags-url [username "default"]) {:as :json})))
+
+(defn update-default-bag
+  [username contents]
+  (:body (http/post (bags-url [username "default"]) {:form-params  contents
+                                                     :content-type :json
+                                                     :as           :json})))
+
+(defn delete-default-bag
+  [username]
+  (:body (http/delete (bags-url [username "default"])))
```

Comment:
This is a nitpick, but you can probably remove the extraction of the `:body` element from the result of `http/delete` since the value is being discarded anyway.